src/Builders/SummaryDictionaryBuilder.cs

```diff
@@ -4,29 +4,42 @@
 
 namespace form_builder.Builders
 {
-    public class SummaryDictionaryBuilder
-    {
+	public class SummaryDictionaryBuilder
+	{
 
-        private Dictionary<string, string> _data = new Dictionary<string,string>();
-        private List<string> _filesData = new List<string>();
+		private Dictionary<string, string> _data = new Dictionary<string, string>();
+		private List<string> _filesData = new List<string>();
 
-        public void Add(string question, string answer, EElementType type)
-        {
-            if (string.IsNullOrWhiteSpace(answer))
-                return;
+		public void Add(string question, string answer, EElementType type)
+		{
+			if (string.IsNullOrWhiteSpace(answer))
+				return;
 
-            if (type == EElementType.FileUpload)
-                _filesData.Add($"{question}: {answer}");
-            else
-                _data.Add(question, answer);
-        }
+			if (type == EElementType.FileUpload)
+				_filesData.Add($"{question}: {answer}");
+			else
+			{
+				if (type == EElementType.Address)
+				{
+					_data.Add("Address", answer);
+				}
+				else if (type == EElementType.Street)
+				{
+					_data.Add("Street", answer);
+				}
+				else
+				{
+					_data.Add(question, answer);
+				}

```

Comment:
i dont think this is needed, the the question value is generated via GetLabelText() on the element, it can be default to Address/Street in there not in the builder.